praetorian_cli/runners/local.py

```
@@ -0,0 +1,296 @@
+
+    if not force and is_installed(tool_name):
+        return get_binary_path(tool_name)
+
+    repo = INSTALLABLE_TOOLS[tool_name]['repo']
+    os_name, arch = _detect_platform()
+
+    os.makedirs(INSTALL_DIR, exist_ok=True)
+
+    # Get latest release asset URL using gh CLI
+    try:
+        result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)